duckduckgo/iOS
# [DuckPlayer] 24. FrontEnd Translation

Task/Issue URL: https://app.asana.com/0/1204099484721401/1208149666429213/f

**Description**:
- Translates Duckplayer FrontEnd and updates copy based on Ship Review
- Removes copy from DP footer in settings
- Updates Copy for DuckPlayer Prompt and Settings to be new, and the same



**Steps to test this PR**:
1. Make yourself internal
2. Add custom config with: https://www.jsonblob.com/api/1276215786342309888
3. Enable Duck Player in ask mode
4. Change app language
5. Go to youtube and Open a video
6. Confirm FE is translated.
7. Go to "Settings" for Duck Player.  
8. Confirm there is no footer in the only cell in the table
9. Check copy for DP modal and settings
